docs/helpers/Eventline.md

```diff
@@ -1,5 +1,9 @@
 # Eventline
 
+Eventline is a doted line which can be used to show so some event occuring at a specific time.
```

Comment:
`dotted`

Reply:
This is not legible at all. Has a lot of spelling mistakes. Please replace this and the line at 5 with this. 
```
`Eventline` is a dotted line (by default) used to show an event occurring at a specific time. 
Unlike `Dateline`, `Eventline` does not have an indicator and therefore is not interactable. Consumers is expected to define the use-case accordingly. 
```

Reply:
I have made the change.